happiness-backend/api/dao/happiness_dao.py

```diff
@@ -64,7 +66,74 @@ def get_happiness_by_group_count(user_ids, page, n):
         Happiness.timestamp.desc()) \
         .paginate(page=page, per_page=n, error_out=False)
 
+def get_happiness_by_filter(user_id, page, per_page, start, end, low, high, query):
+    """
+    Filters according to the provided arguments. Checks to see what filters to apply. Will not apply the following
+    filters with the following arguments
+     low/high = 11
```

Comment:
This seems like a very arbitrary choice for a number to not apply the filters. Shouldn't it not apply if low or high is `None`? Why 11? Also will the filters apply if low > high? The docs should specify behavior in this case. Same thing with start > end

Reply:
i chose 11 bc happiness entries cannot be a value 11, but i understand its arbritrary, do you think chaging the default value to None would be better? i also agree about the specified behavior, so im changing it so itll raise exceptions if low>high or start ">" end, thanks!